.github/workflows/receive-fork-review.yml

```diff
@@ -0,0 +1,28 @@
+name: Receive Fork Review
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  approved:
+    # only run the workflow on forks and when a PR is reviewed and approved
+    if: |
+      (github.event.pull_request.head.repo.full_name != github.repository) &&
+      (github.event.review.state == 'approved')
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "This PR is approved and e2e tests will begin running."
+  build:
```

Comment:
Wondering if we can combine the two jobs into one, since 

(1) these two jobs are fairly simple and 
(2) so that we can remove latency between the two.